docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "2"
+services:
+  build-test:
+    build:
+      context: ./ops/ci
+      dockerfile: Dockerfile.node-alpine_with_glibc
+    command: |
+      sh -c "sh -s <<'EOF'
+      set -ex
+      yarn install
+      yarn validate
+      EOF"
+    volumes:
+      - .:/root/app
+    working_dir: /root/app
+    environment:
+      - YARN_CACHE_FOLDER=/yarn_home/.cache
+      - HOME=/yarn_home
+      - NPM_TOKEN
+      - GO_PIPELINE_COUNTER
+  build-publish:
+    build:
+      context: ./ops/ci
+      dockerfile: Dockerfile.node-alpine_with_glibc
```

```suggestion
    image: node:12
```
That said, if it's possible to bump this above 12 - ideally as high as 16 - that would be great. 12 is already in maintenance LTS and will be out of support in 6mo: https://nodejs.org/en/about/releases/